server.go

```diff
@@ -262,6 +276,14 @@ func (s *Server) processUnaryRPC(t transport.ServerTransport, stream *transport.
 			}
 		}()
 	}
+	monitor := s.opts.serverMonitor.NewServerMonitor(monitoring.Unary, stream.Method())
+	defer func() {
```

Comment:
Curious: Have you benchmarked for throughput the overhead of the additional defer call on this procedure and elsewhere?